spring-kafka/src/main/java/org/springframework/kafka/config/SimpleKafkaListenerEndpoint.java

```diff
@@ -1,71 +0,0 @@
-/*
- * Copyright 2016 the original author or authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.springframework.kafka.config;
-
-
-import org.springframework.kafka.listener.MessageListener;
-import org.springframework.kafka.listener.MessageListenerContainer;
-import org.springframework.kafka.support.converter.MessageConverter;
-
-/**
- * A {@link KafkaListenerEndpoint} simply providing the {@link MessageListener} to
- * invoke to process an incoming message for this endpoint.
- *
- * @param <K> the key type.
- * @param <V> the value type.
- *
- * @author Stephane Nicoll
- * @author Gary Russell
- */
-public class SimpleKafkaListenerEndpoint<K, V> extends AbstractKafkaListenerEndpoint<K, V> {
```

Comment:
It's not used anywhere, not documented, and I don't see it adding any real value. Why go through conversion from `ConsumerRecord` to `Message<?>` and then immediately discard the conversion?